src/main/kotlin/org/oooc/kry/board/web/controller/ArticleVoteController.kt

```diff
@@ -9,8 +11,10 @@ class ArticleVoteController(val articleVoteService: ArticleVoteService) {
 
     // CREATE ARTICLE_VOTE
     @PostMapping("/{boardName}/article/{articleNo}/vote")
-    fun createArticleVote(@PathVariable boardName: String, @PathVariable articleNo: Long, @RequestParam updown: Byte) {
-
+    fun createArticleVote(@PathVariable boardName: String,
+                          @PathVariable articleNo: Long,
+                          @RequestBody articleVoteCreateRequestDTO: ArticleVoteCreateRequestDTO) {
+        articleVoteService.createArticleVote(boardName, articleNo, articleVoteCreateRequestDTO.updown)

```

Comment:
```suggestion
    fun createArticleVote(
        @PathVariable boardName: String,
        @PathVariable articleNo: Long,
        @RequestBody articleVoteCreateRequestDTO: ArticleVoteCreateRequestDTO
    ) {
        articleVoteService.createArticleVote(boardName, articleNo, articleVoteCreateRequestDTO.updown)

```
@berry2971 요게 좀 더 보기 좋을 것 같은데, 어떻게 생각하세요?
현재대로라면 메서드도 길고 인자명도 길 때 보기 안 좋을 것 같아서요.

Reply:
@dsyun96 제안해주신 스타일이 더 깔끔해 보여서, 그렇게 수정했습니다. 감사합니다.